printer_mods/dankeeling/CreativeLiveCameraMount/.metadata.yml

```diff
@@ -0,0 +1,31 @@
+---
+# The title of your mod. Choose a short and descriptive name
+title: Creative Live! Camera Mount
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: A mod to provide a mount for the Creative Live! Camera that mounts on 20mm extrusion.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: Homelabhobbyist V2.4975
```

Comment:
Ok, I think this is the last thing:  `discord_username` should be the username, not the display name.  I think yours is `klest?`

Reply:
@MapleLeafMakers I agree with your assessment -- any objection to proposing that as a suggestion so DanKeeling can commit it directly?

Reply:
Made the update!